frontend/src/components/FormControl/FormLabel/FormLabel.tsx

```diff
@@ -95,6 +95,7 @@ export const FormLabel = ({
           >
             <Icon
               ml="0.5rem"
+              mb="0.1rem"
```

Comment:
We usually don't use `0.1rem`. Is there an issue with the bottom margin on this?

Reply:
Without it, the tooltip icon appears 1-2px too low. Perhaps there is a better way to resolve this